stable/appmesh-controller/README.md

```diff
@@ -387,7 +387,7 @@ Parameter | Description | Default
 `sidecar.logLevel` | Envoy log level | `info`
 `sidecar.envoyAdminAccessPort` | Envoy Admin Access Port | `9901`
 `sidecar.envoyAdminAccessLogFile` | Envoy Admin Access Log File | `/tmp/envoy_admin_access.log`
-`sidecar.resources.requests` | Envoy container resource requests | `requests: cpu 10m memory 32Mi`
+`sidecar.resources.requests` | Envoy container resource requests | `requests: cpu 512 memory 64Mi`
```

Comment:
So this is where ECS and Kubernetes differ. ECS equates 1024 CPU units to 1 AWS vCPU, whereas Kubernetes uses 1 cpu as 1 vCPU.

More about CPU units for k8s in the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#cpu-units).

Reply:
This needs to be changed to something like `500m` or `0.5`